inbox/models/category.py

```diff
@@ -18,6 +19,14 @@
 EPOCH = datetime.utcfromtimestamp(0)
 
 
+class CategoryNameType(String):
```

Comment:
Can you put a docstring on this describing what it's for? I think this would make it more clear to future readers!